keps/sig-node/4742-node-labels-downward/README.md

```diff
@@ -0,0 +1,812 @@
+Signoff checklist to be completed.
+
+Check these off as they are completed for the Release Team to track. These
+checklist items _must_ be updated for the enhancement to be released.
+-->
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input (including test refactors)
+  - [ ] e2e Tests for all Beta API Operations (endpoints)
+  - [ ] (R) Ensure GA e2e tests meet requirements for [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
+  - [ ] (R) Minimum Two Week Window for GA e2e tests to prove flake free
+- [ ] (R) Graduation criteria is in place
+  - [ ] (R) [all GA Endpoints](https://github.com/kubernetes/community/pull/1806) must be hit by [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+Exposing node topology labels through the downward API provides a native method
+from which to attain node label values in pods without the need of a sidecar
+to extract information.
+
+## Motivation
+
+We’d like to change the runtime behavior of containers based on node labels.
+
+A number of other use cases exist for providing node labels to pods. One
+example is utilizing topology data from cloud providers, which are automatically
+assigned to nodes, and then used to tailor workloads to minimize cross-zone data
+transfers and reduce costs. Having an easy way for pods to access these node
+topology labels would provide users a straightforward, maintainable way to
+optimize their workloads given topology constraints.
+
+Workarounds today typically involve using an initContainer to query the
+Kubernetes API and then pass data via shared volume to other containers within
```

Comment:
another alternative is a webhook that will inject those values. It will be great to fill out the alternatives section to list out pros and cons of each and why we need this one

Reply:
Added this as one of the alternative solutions in the motivations section, thanks!